src/components/Authentication/Signup.jsx

```diff
@@ -0,0 +1,124 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useAuth } from '../../context';
+
+import './Signin-Signup.css';
+
+const SignUp = () => {
+    const { signUpHandler } = useAuth();
+
+    const [userInput, setUserInput] = useState({
+        firstName: '',
+        lastName: '',
+        email: '',
+        password: '',
+    });
+
+    return (
+        // <!-- Login body -->
+
+        <div className="login-container">
+            <div className="login">
+                <p className="login-header">Sign Up</p>
+
+                <form
+                    className="login-input-form"
+                    onSubmit={(event) => signUpHandler(event, userInput)}
```

Comment:
You can preventDefault here as well know need to pass whole event to a function.

Reply:
Noted. Thank you!